packages/web/src/components/spa/HackAUBG/VideoSection/video_section.css

```diff
@@ -24,20 +27,28 @@
     justify-content: center;
     align-items: center;
     display: flex;
-    padding: 3% 0;
     padding: 3% 0%;
     text-align: justify;
 }
 
 @media only screen and (max-width: 900px) {
+    .video-container {
+        height: 90vh;
```

Comment:
This height for mobile is a little bit too much. There is too much space at the top and bottom part between the video section and the other sections.